### PR TITLE
Sga 126 log own exercise session

### PR DESCRIPTION
**Pull request by ugnspa**
## PR Checklist

- [x] After user ends his exercise session calories, distance, exercise type and map is written to health connect
- [x] Removed pause/resume buttons because health connect only lets to write continuous route
- [x] Exercise session is printed to exercise screen (the numbers of unit stays the same if FitFlow app is prioritized first in health connect)

### CODE REVIEW

File: app/src/main/java/com/github/k409/fitflow/ui/screen/activity/exerciseSession/ExerciseSessionScreen.kt [hunk 1]
```diff
@@ -142,13 +136,13 @@ fun ExerciseSessionScreen(
             Spacer(modifier = Modifier.weight(1f))
             ToSettings()
         }
-        if (showConfirmationDialog) {
+        if (showConfirmationDialogStart) {
             ConfirmDialog(
                 dialogTitle = "Are you sure you want start this exercise session?",
```
Comment (paulius-mongirdas) [inline app/src/main/java/com/github/k409/fitflow/ui/screen/activity/exerciseSession/ExerciseSessionScreen.kt]: UI strings should be moved to strings.xml

**Review by paulius-mongirdas — COMMENTED** [review]
Metrics here could be displayed in a more user-friendly way. Maybe add labels like distance, pace, calories.

File: app/src/main/java/com/github/k409/fitflow/ui/screen/activity/exerciseSession/ExerciseSessionScreen.kt (hunk 1 above)
Comment (ugnspa) [reply app/src/main/java/com/github/k409/fitflow/ui/screen/activity/exerciseSession/ExerciseSessionScreen.kt]: I have a task for Exercise session UI during this sprint

Comment (paulius-mongirdas) [reply app/src/main/java/com/github/k409/fitflow/ui/screen/activity/exerciseSession/ExerciseSessionScreen.kt]: All good then. Maybe this feedback will be useful in that task.

**Review by paulius-mongirdas — APPROVED** [review]

**Review by mantasjasikenas — APPROVED** [review]

**Pull request merged**